src/software/Localization/convertDescFloat2Uchar.cpp

```diff
@@ -2,10 +2,13 @@
 #include <openMVG/logger.hpp>
 
 #include <boost/progress.hpp>
-#include <boost/filesystem.hpp>
 #include <boost/program_options.hpp> 
 #include <boost/algorithm/string/case_conv.hpp> 
 
+#define BOOST_NO_CXX11_SCOPED_ENUMS
```

Comment:
Yes, it's difficult to know how it is compiled in the system packages.
I will try to improve the description of this build option and add a sentence about this in the INSTALL file. And I will merge only the first commit of the PR, if it works for you @zvrba with this cmake build option.